src/app/shared/issue/duplicateOf/duplicate-of.component.ts

```diff
@@ -118,6 +140,20 @@ export class DuplicateOfComponent implements OnInit {
     return clone;
   }
 
+  private changeFilter(issuesObservable: Observable<Issue[]>, searchInputString): Observable<Issue[]> {
+    return issuesObservable.pipe(
+      first(),
+      map((issues) => {
+        return applySearchFilter(
+          searchInputString,
+          [TABLE_COLUMNS.ID, TABLE_COLUMNS.DUPLICATED_ISSUES, TABLE_COLUMNS.TITLE],
```

Comment:
i'm guessing you included the `TABLE_COLUMNS.DUPLICATED_ISSUES` to search through so that we can find the "root" issue of a duplicate? e.g. this screenshot below:
<img width="754" alt="Screenshot 2022-03-17 at 9 56 51 AM" src="https://user-images.githubusercontent.com/68138671/158721453-01bd8a83-4d39-41f0-abc9-e6c4644d9b7b.png">
 
this only works with issue number (since that's what's stored in the duplicated issues column):

<img width="759" alt="Screenshot 2022-03-17 at 10 04 25 AM" src="https://user-images.githubusercontent.com/68138671/158721946-d0ffbe86-d20d-4ac9-a5a3-5625bfa6f6d0.png">

is this behaviour intended?

Reply:
I see, thanks for pointing it out. It should not happen so I will remove `TABLE_COLUMNS.DUPLICATED_ISSUES`.